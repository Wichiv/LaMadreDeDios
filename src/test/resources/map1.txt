C - 3 - 3
M - 1 - 1
A - Dumbo - 0 - 0 - S - GAAADADAGA
T - 2 - 2 - 1