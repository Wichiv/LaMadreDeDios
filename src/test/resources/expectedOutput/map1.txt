C - 3 - 3
M - 1 - 1
# {A comme Aventurier} - {Nom de l’aventurier} - {Axe horizontal} - {Axe vertical} - {Orientation} - {Nb. trésors ramassés}
A - Dumbo - 2 - 2 - S - 1